### ISSUE TITLE
Lower hashes with same config from 2.4.3 to 2.4.4. About 85 H/s per AMD Vega FE

### ISSUE THREAD

**Issue by jam3972** [fireice-uk/xmr-stak#1641]
I am also getting a hash drop between versions. Using 3x AMD VEGA FE only. No CPU.
2.4.3 I was getting 6255 H/s with this config.
2.4.4 I get about 6000 H/s with this config.

looking at the Hashrate report, it is an even drop across all cards/threads.

I also tried auto config of amd.txt and it was even slower.
Same behavior with pre-made binaries from release page and with self-compiled. Win 10 x64 ver 1803.

Using the following amd.txt for both versions.

"gpu_threads_conf" : [
  // gpu: gfx901 memory:15822
  // compute units: 64
  { "index" : 0,
    "intensity" : 2016, "worksize" : 8,
    "affine_to_cpu" : false, "strided_index" : 1, "mem_chunk" : 2,
    "comp_mode" : true
  },
  { "index" : 0,
    "intensity" : 2016, "worksize" : 8,
    "affine_to_cpu" : false, "strided_index" : 1, "mem_chunk" : 2,
    "comp_mode" : true
  },
  // gpu: gfx901 memory:15822
  // compute units: 64
  { "index" : 1,
    "intensity" : 2016, "worksize" : 8,
    "affine_to_cpu" : false, "strided_index" : 1, "mem_chunk" : 2,
    "comp_mode" : true
  },
  { "index" : 1,
    "intensity" : 2016, "worksize" : 8,
    "affine_to_cpu" : false, "strided_index" : 1, "mem_chunk" : 2,
    "comp_mode" : true
  },
  // gpu: gfx901 memory:15822
  // compute units: 64
  { "index" : 2,
    "intensity" : 2016, "worksize" : 8,
    "affine_to_cpu" : false, "strided_index" : 1, "mem_chunk" : 2,
    "comp_mode" : true
  },
  { "index" : 2,
    "intensity" : 2016, "worksize" : 8,
    "affine_to_cpu" : false, "strided_index" : 1, "mem_chunk" : 2,
    "comp_mode" : true
  },

],

here is my config.txt for both:

"call_timeout" : 10,
"retry_time" : 30,
"giveup_limit" : 0,
"verbose_level" : 3,
"print_motd" : true,
"h_print_time" : 60,
"aes_override" : null,
"use_slow_memory" : "warn",
"tls_secure_algo" : true,
"daemon_mode" : false,
"flush_stdout" : false,
"output_file" : "",
"httpd_port": 3336,
"http_login" : "",
"http_pass" : "",
"prefer_ipv4" : true,

**Comment by JerichoJones** [comment]
Did you upgrade to 1803? If so, did you remove and re-install the drivers?

**Comment by jam3972** [comment]
Yes and Yes. Neither version worked until I re-installed drivers.
After re-installing drivers, 2.4.3 worked with sames speeds, but 2.4.4 has this issue.

**Comment by jhb9** [comment]
Just tried 2.4.5 coming from 2.4.3 and I am seeing a drop in hashes on my single Vega56.  Just digging into it..

**Comment by jhb9** [comment]
Just an update using DDU to remove and reinstall the drivers appears to have worked for me.

**Comment by JerichoJones** [comment]
@jam3972 Did you use DDU to remove the drivers?
